index.html

```diff
@@ -1851,76 +1770,72 @@ <h3> 'Subject Only' Terms of Use </h3>
 
        </section>
 
-  <section>
-   <h3> Passing on a Verifiable Credential </h3>
 
-      <p class="issue" data-number=204>
-The group is currently debating the best security model for delegation. Readers
-should treat this entire section as at-risk and currently under debate.
-      <p>
+    </section>
 
-   <p>
+    <section>
+      <h2>Credential Uniquely Identifies Subject</h2>
 
-When the subject passes a verifiable credential to a
-holder, the subject SHOULD issue a new verifiable credential to the holder in which:
-the issuer is the subject,
-the subject is the holder to whom the verifiable credential is being passed,
-and the claim contains the properties that are being passed on. In addition, the
-holder creates a verifiable presentation that contains these two
-verifiable credentials.
+      <p>
+In this case, the <a>claim</a> may contain multiple properties that each
+provide an aspect of the identity of the subject, and which together
+unambiguously identify the subject. 
 </p>
```

Comment:
The above change was not made by myself, but was already in the master copy that I pulled